src/sentry/models/dsymfile.py

```diff
@@ -30,6 +31,86 @@
 }
 
 
+'''
+system symbols:
+
+  architecture   VARCHAR(20)
+  sdk-version    MAJOR.MINOR.PATCHLEVEL-BUILD
+  uuid           UUID
+  path           VARCHAR(255)
+  address        BIGINT
+  symbol         TEXT
+
+lookup logic:
+
+  primary lookup:
+    uuid -> exact match
+    address -> lower than or equal to reference address
+      limit 1
+
+  secondary lookup:
+    path -> exact match
+    sdk-version -> fuzzy match
+    address -> lower than or equal to reference address
+      limit 1
+'''
+
+
+class DSymSDK(Model):
+    __core__ = False
+    dsym_type = models.CharField(max_length=20, db_index=True)
+    sdk_name = models.CharField(max_length=20)
+    version_major = models.IntegerField()
+    version_minor = models.IntegerField()
+    version_patchlevel = models.IntegerField()
+    version_build = models.CharField(max_length=40)
+
+    class Meta:
+        app_label = 'sentry'
+        db_table = 'sentry_dsymsdk'
+        index_together = [
+            ('version_major', 'version_minor', 'version_patchlevel',
+             'version_build'),
+        ]
+
+
+class DSymObject(Model):
+    __core__ = False
+    cpu_name = models.CharField(max_length=40)
```

Comment:
I'm generally quite worried about indexes over things that are not very unique. Eg: in practice for a long time we will only have two cpu architectures. That will be quite a useless index because it only cuts the search space into two.